### PR TITLE
Add a new optional float to GD_TIME to specify a max applied time

### PR DESCRIPTION
**Pull request by wortiz**
Adds a new float option to the end of GD_TIME to allow specifying a maximum time

This allows you to use GD_TIME to ramp up to a fixed value.

### CODE REVIEW

**Comment by acochrane** [comment]
.runguts

File: src/mm_input_util.c [hunk 1]
```diff
@@ -1929,3 +1929,39 @@ strip(char string[])
 /*****************************************************************************/
 /*****************************************************************************/
 /*****************************************************************************/
+
+/* Read up to n doubles from a file up to a newline 
+
+   ifp is the file ptr where the current line should be looked at
+
+   n is the number of doubles to look for
```
Comment (acochrane) [inline src/mm_input_util.c]: What if you look for 4 doubles and there are only 3?

Doesn't read_constants (mm_input.c:10892) do this without specifying the length and automatically allocating the right amount of space?

**Comment by natis1** [comment]
.runguts

**Comment by natis1** [comment]
I'm investigating the goma test bot right now.

**Comment by wortiz** [comment]
.runguts

**Pull request merged**